src/Pages/ALLmeeting.tsx

```diff
@@ -6,8 +6,13 @@ import MeetingDetails from '../Components/MeetingDetails'
 const ALLmeeting: React.FC = () => {
   const [selectedMeetingId, setSelectedMeetingId] = useState<number | null>(null);
   return (
+<<<<<<< HEAD
+  <Link to={"/ALLmeeting"}>
+    <div className='w-[100%] bg-blue-600/20'>
+=======
   
     <div className='w-[100%] bg-transparent'>
+>>>>>>> 48979b422e03693f5529f759fa31c9a935b254a2
```

Comment:
Looks like there was a merge conflict here and you attempted to solve this conflict. Kindly leave all future merge conflicts for me to solve.
Also, please edit this file to what it was initially because this flags an error and hinders the pages from being rendered.